content/en/blog/_posts/2017-01-00-Running-Mongodb-On-Kubernetes-With-Statefulsets.md

```diff
@@ -201,91 +189,52 @@ The configuration for the StatefulSet looks like this:
 
 
 ```
-apiVersion: apps/v1beta1
-
+apiVersion: apps/v1
```

Comment:
This is a bit anachronistic: This is a blog article, and blog articles are tied to a publication date. StatefulSet in _apps/v1beta1_ sounds right for an article from January 2017. Does _apps/v1_ definitely work on a cluster from that era?

Instead, how about adding a comment somewhere in the article explaining that it's no longer current?

Reply:
Happy to revert the changes, but since this is the only MongoDB article I found on kubernetes.io. Alternatively, would it be of interest to have an updated one?

Reply:
I just noticed: the MongoDB container doesn't work with the parameters provided at this time. That would need to be updated too. (Sorry I didn't add those changes.)

Reply:
I agree with @sftim - I'd just fix the formatting instead of updating it. In general we only update the content of things that area year or less old.

Reply:
Thank you for the feedback, I will update the PR accordingly ~end of week. Are you interested in a new post or documentation how to do this with a current API and MongoDB?

Reply:
@sftim @mrbobbytables thank you for your patience. I have updated the PR to follow your advice and I have also reformatted the file to make it easier to edit in the future. Please let me know if I can do anything else to improve this once.